Training/Data/Network/Request/GetRepoRequest.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  GetReposRequest.swift
+//  Training
+//
+//  Created by Ong Yue Huei on 09/07/2021.
+//
+
+import Alamofire
+
+struct GetRepoRequest: BaseRequest {
+    typealias Response = Repo
+    var path: String = "/repos/"
+    var method: HTTPMethod = .get
+
+    init(ownerRepo: String) {
+        path += ownerRepo
+    }
```

Comment:
[nits]
可能であれば`var`(mutable)よりも`let`(immutable)で宣言したほうが良いです！
```suggestion
    let path: String
    let method: HTTPMethod = .get

    init(ownerRepo: String) {
        path = "/repos/\(ownerRepo)"
    }
```

Reply:
またはこの様な形

```suggestion
    var path: String { "/repos/\(ownerRepo)" }
    let method: HTTPMethod = .get
    let ownerRepo: String

    init(ownerRepo: String) {
        self.ownerRepo = ownerRepo
    }
```

Reply:
はい！分かりました！教えていただきありがとうございます！